app/assets/javascripts/hydra-editor/keyboard_support.js

```diff
@@ -1,14 +0,0 @@
-Blacklight.onLoad(function() {
-  $('body').on('keypress', '.multi-text-field', function(event) {
-    var $activeField = $(event.target).parents('.field-wrapper'),
-        $activeFieldControls = $activeField.children('.field-controls'),
-        $addControl = $activeFieldControls.children('.add'),
-        $removeControl = $activeFieldControls.children('.remove');
-    if (event.keyCode == 13) {
-      event.preventDefault();
-      $addControl.click()
-      $removeControl.click()
-    }
-  });
-});
-
```

Comment:
I'm wondering what the preferred way would be to submit the form if ENTER were to default to the buttons.  I'd think they would want some kind of option to submit in a single keystroke, but maybe I'm just nuts here.  Or maybe there are assistive tools that render it a non-issue...?  As a mostly-but-not-100%-keyboard user, I can only comment on people like me who *prefer* the keyboard, but aren't forced to use it exclusively.